velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -48,6 +48,28 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
     "order by c1 desc nulls first, c0 asc nulls first",
 };
 
+/// Common set of window function over clauses for window frame tests to ensure
+/// total ordering for deterministic results in ROW mode.
+static std::vector<std::string> kFrameOverClauses = {
+    "partition by c0 order by c1, c2",
+    "partition by c1 order by c0, c2",
+    // No partition by clause.
+    "order by c0 asc, c1 desc, c2",
+    "order by c1 asc, c0 desc, c2",
+    // No order by clause.
+    "partition by c0, c1, c2",
+};
+
+/// Common set of window function frame clauses with current row, unbounded
```

Comment:
Why is this common set ? Maybe call this kRangeFrameClauses.

Reply:
Thanks for the suggestion, I had thought of adding the distinction with the row frames PR. Modified accordingly!